modules/ztp-removing-content-from-managed-clusters.adoc

```diff
@@ -0,0 +1,132 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp_far_edge/ztp-configuring-managed-clusters-policies.adoc
+
+:_content-type: PROCEDURE
+[id="ztp-removing-content-from-managed-clusters_{context}"]
+= Changing applied managed cluster CRs using policies
+
+You can remove content from a custom resource (CR) deployed to a managed cluster through a policy.
+
+By default, all `Policy` CRs created from a `PolicyGenTemplate` CR have a `complianceType` set to `musthave`.
+In this mode, when you remove a field from a CR, the value is removed from the policy but is not removed from the CR on the managed cluster.
+A `musthave` policy without the removed field is still compliant because the CR on the managed cluster has all the specified content.
+
+To remove the content from the managed cluster, set the `complianceType` field to `mustonlyhave`.
+In this mode, the policy ensures that the CR on the cluster is an exact match of what is specified in the policy.
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (`oc`).
+
+* You have logged in to the hub cluster as a user with `cluster-admin` privileges.
+
+* You have deployed a managed cluster from a hub cluster running {rh-rhacm}.
+
+* You have installed {cgu-operator-full} on the hub cluster.
+
+.Procedure
+
+. Remove the content that you no longer need from the affected CRs. In this example, the `disableDrain: false` line is removed from the `SriovOperatorConfig` CR.
++
+.Example CR
+
+[source,yaml]
+----
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovOperatorConfig
+metadata:
+  name: default
+  namespace: openshift-sriov-network-operator
+  annotations:
+    ran.openshift.io/ztp-deploy-wave: "10"
+spec:
+  configDaemonNodeSelector:
+    "node-role.kubernetes.io/$mcp": ""
+  disableDrain: false <1>
+  enableInjector: true
+  enableOperatorWebhook: true
+----
+<1> Remove the `disableDrain: false` line.
+
+. Change the `complianceType` of the affected policies to `mustonlyhave` in the `common-ranGen.yaml` file.
++
+.Example YAML
+[source,yaml]
+----
+ -  fileName: SriovSubscriptionNS.yaml
+    policyName: "subscriptions-policy"
+    complianceType: mustonlyhave
+ -  fileName: SriovSubscriptionOperGroup.yaml
+    policyName: "subscriptions-policy"
+    complianceType: mustonlyhave
+ -  fileName: SriovSubscription.yaml
+    policyName: "subscriptions-policy"
+    complianceType: mustonlyhave
+----
```

Comment:
This should be the group-du-sno-rangen.yaml PGT. That is where the SriovOperatorConfig CR is included. In there only that one entry needs the mustonlyhave.